sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -45,6 +46,7 @@ public class MetricDataMapper {
 
     private static final Set<String> OTEL_PRE_AGGREGATED_STANDARD_METRIC_NAMES = new HashSet<>(4);
     private static final List<String> EXCLUDED_METRIC_NAMES = new ArrayList<>();
+    public static final AttributeKey<String> APPLICATIONINSIGHTS_INTERNAL_METRIC_NAME = AttributeKey.stringKey("applicationinsights_internal_metric_name");
```

Comment:
(follows attribute naming conventions a bit more)
```suggestion
    public static final AttributeKey<String> APPLICATIONINSIGHTS_INTERNAL_METRIC_NAME = AttributeKey.stringKey("applicationinsights.internal.metric_name");
```